### DISCUSSION TITLE
Body Hash Did Not Verify Problem

### DISCUSSION THREAD

**Discussion by TR-SametOzen** [postalserver/postal#3193] [Development discussions]
Hello,

There is a "Body Hash Did Not Verify" problem when verifying DKIM. How can I solve this?

<img width="1432" alt="image" src="https://github.com/user-attachments/assets/a56782f7-324a-460e-918e-2e087282cb2a">

**Comment by willpower232** [comment]
Just to double check, what does https://www.mail-tester.com/ say about it?

**Reply by TR-SametOzen** [reply]
Hello, @willpower232,

Rating here 10/10. It falls into INBOX in Gmail. Yandex drops to INBOX. But Outlook Spam. How can I fix this?

**Reply by willpower232** [reply]
If you have a 10/10 from mail-tester then it must be Microsoft specific, I'm not sure what else you can do.
